transport/src/main/java/io/netty/channel/EventLoopGroup.java

```diff
@@ -49,4 +49,10 @@
      */
     @Deprecated
     ChannelFuture register(Channel channel, ChannelPromise promise);
+
+    /**
+     * Return the defaults {@link ChannelFactory} for channels to be created in the context of this group
+     * @return a default channel factory or null if not provided
+     */
+    ChannelFactory defaultChannelFactory(boolean serverSide);
```

Comment:
@eolivelli we can not do this for now as this would be a breaking change (adding a method to an interface